bin/build-metadata.js

```diff
@@ -214,6 +214,18 @@ function removeAllNumberKeys( obj ) {
 	return _.omitBy( obj, function( val, key ) { return /^\d+$/.test( key ); } );
 }
 
+function removeRegionCodeAndCountryDialCodeIfSameWithCountryDialCode( countryData ) {
+	for ( var key in countryData ) {
+		if ( countryData.hasOwnProperty( key ) ) {
+			if ( ! countryData[ key ].regionCode && ( countryData[ key ].countryDialCode + countryData[ key ].regionCode ) === countryData[ key ].dialCode ) {
```

Comment:
Could be written more concisely if we used destructuring and dropped the second `regionCode` mention (since it will always be empty at that point):
```
const { regionCode, countryDialCode, dialCode } = country;
if ( ! regionCode && countryDialCode === dialCode ) {
    // ...
}
```